connector_importer/utils/mapper_utils.py

```diff
@@ -284,11 +276,37 @@ def modifier(self, record, to_attr):
             # override by param
             search_operator = modifier.search_operator
 
-        # finally search it
         search_args = [(modifier.search_field, search_operator, search_value)]
-
         value = rel_model.search(search_args)
 
+        value = _handle_missing_values(
```

Comment:
can you please split this change to its own commit so that we can backport it?

Reply:
You can apply these changes on v16 if it eases your life and then we cherry-pick the commit  ;)

Reply:
Thanks @simahawk ,
- I don't have plans to make a PR for v17. And I've isolated the modifier change out so you can cherry-pick it easily.
- Would you like me to create a PR to apply it to version 16?

Reply:
> Thanks @simahawk ,
> 
>     * I don't have plans to make a PR for v17. And I've isolated the modifier change out so you can cherry-pick it easily.

cool, tnx

>     * Would you like me to create a PR to apply it to version 16?

yes please, it will serve as a reminder :pray: